packages/react/src/components/IconButton/next/IconButton.stories.js

```diff
@@ -35,14 +35,16 @@ export default {
 const PlaygroundStory = (props) => {
   const { align, defaultOpen, disabled, kind, label } = props;
   return (
-    <IconButton
-      align={align}
-      defaultOpen={defaultOpen}
-      disabled={disabled}
-      kind={kind}
-      label={label}>
-      <Edit />
-    </IconButton>
+    <div style={{ margin: '3rem' }}>
+      <IconButton
+        align={align}
+        defaultOpen={defaultOpen}
+        disabled={disabled}
+        kind={kind}
+        label={label}>
```

Comment:
I think this needs to have the `size` prop added to it? Not sure,  the size controls weren't working for me and I think that might be why

Reply:
Ah sorry, I was thinking of this bit: `test the playground knobs for size & kind` and thought that meant the controls. You can totally ignore this then!